ibis/streamlit/__init__.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import annotations
+
+from typing import Any
+
+from streamlit.connections import ExperimentalBaseConnection
+from streamlit.runtime.caching import cache_data
+
+import ibis
+from ibis.backends.base import BaseBackend
+
+__all__ = ["IbisConnection"]
+
+
+class IbisConnection(ExperimentalBaseConnection[BaseBackend]):
+    def _connect(self, **kwargs) -> BaseBackend:
```

Comment:
A tiny docstring showing an example streamlit config (and what options are supported) would be useful in understanding this bit of code IMO.

Reply:
Done!